### PR TITLE
Merge v1-dev into v1-master

### PR DESCRIPTION
**Pull request by kaderhashmi**

### CODE REVIEW

File: DFC.Api.JobProfiles.Data/DFC.Api.JobProfiles.Data.csproj [hunk 1]
```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <CodeAnalysisRuleSet>../DFC.Digital.CodeAnalysis.ruleset</CodeAnalysisRuleSet>
+    <LangVersion>latest</LangVersion>
```
Comment (thelious1974) [inline DFC.Api.JobProfiles.Data/DFC.Api.JobProfiles.Data.csproj]: This is missing it's ProjectGuid field,  which SonarCloud requires.

File: DFC.Api.JobProfiles.ProfileServices/DFC.Api.JobProfiles.ProfileServices.csproj [hunk 2]
```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <CodeAnalysisRuleSet>../DFC.Digital.CodeAnalysis.ruleset</CodeAnalysisRuleSet>
+    <LangVersion>latest</LangVersion>
```
Comment (thelious1974) [inline DFC.Api.JobProfiles.ProfileServices/DFC.Api.JobProfiles.ProfileServices.csproj]: This is missing it's ProjectGuid field, which SonarCloud requires.

File: DFC.Api.JobProfiles.ProfileServicesTests/DFC.Api.JobProfiles.ProfileServicesTests.csproj [hunk 3]
```diff
@@ -0,0 +1,41 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <CodeAnalysisRuleSet>../UnitTests.CodeAnalysis.ruleset</CodeAnalysisRuleSet>
+    <IsPackable>false</IsPackable>
+    <LangVersion>latest</LangVersion>
```
Comment (thelious1974) [inline DFC.Api.JobProfiles.ProfileServicesTests/DFC.Api.JobProfiles.ProfileServicesTests.csproj]: This is missing it's ProjectGuid field, which SonarCloud requires.

File: DFC.Api.JobProfiles.Repository.CosmosDb/DFC.Api.JobProfiles.Repository.CosmosDb.csproj [hunk 4]
```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <CodeAnalysisRuleSet>../DFC.Digital.CodeAnalysis.ruleset</CodeAnalysisRuleSet>
+    <LangVersion>latest</LangVersion>
```
Comment (thelious1974) [inline DFC.Api.JobProfiles.Repository.CosmosDb/DFC.Api.JobProfiles.Repository.CosmosDb.csproj]: This is missing it's ProjectGuid field, which SonarCloud requires.

**Review by Muthuramana — APPROVED** [review]

**Pull request merged**